app/helpers/api/v2/accounts/reports_helper.rb

```diff
@@ -27,6 +27,32 @@ def generate_labels_report
     end
   end
 
+  def generate_conversations_heatmap_report
+    report_params = {
+      type: :account,
+      group_by: 'hour',
+      since: params[:since],
+      until: params[:until],
+      metric: 'conversations_count',
+      timezone_offset: '5.5',
+      business_hours: false
+    }
+    data = V2::ReportBuilder.new(Current.account, report_params).build
+
+    # data format is { timestamp: 1231242342, value: 3}
+    # we need to convert it to { date: "2020-01-01", hour: 12, value: 3}
+    #
+    # since the data from the report builder is already timezone aware,
+    # the conversion to date and hour is done without timezone
+    # rubocop:disable Rails/TimeZone
+    data.map do |d|
+      date = Time.at(d[:timestamp]).to_date
+      hour = Time.at(d[:timestamp]).hour
+      [date, hour, d[:value]]
+    end
+    # rubocop:enable Rails/TimeZone
```

Comment:
@sojan-official need your thoughts on disabling this rule, I've added the reasons in the comments, is there a better way to do this without `Time.at`

Reply:
@scmmishra  The comment makes sense to me. But if we can get through with using `Time.zone.at` or `Time.at.in_time_zone` ref: https://www.rubydoc.info/gems/rubocop/0.57.2/RuboCop/Cop/Rails/TimeZone 

It would be better if we could get rid of the comment as well as disabling the rule.

Reply:
I tried using `Time.zone.at` but it would apply the timezone calculation to already existing timezone adjusted value, so the 12:00 AM records show up in 5:30 AM records